apps/back/src/Expense.php

```diff
@@ -0,0 +1,68 @@
+<?php
```

Comment:
Same comment for this class and User, you could move those files to a repository like `App\Model`.
Do not forget to add a namespace